poker.cs

```diff
@@ -4,125 +4,118 @@
+        if (p8 != Bytes.EMPTY) Players.put(len++, p8);
+        Players.put(-1, new Bytes(Convert.ToByte(len)));
     }
-    public void dealPublicCard(int card)
+    public void DealPublicCard(int card)
     {
-        int len = table_cards.getDefault(-1, 0);
-        table_cards.put(len++, card);
-        table_cards.put(-1, len);
+        int len = TableCards.getDefault(-1, 0);
+        TableCards.put(len++, card);
+        TableCards.put(-1, len);
     }
-    public void dealPrivateCard(int player, Bytes cardHash)
+    public void DealPrivateCard(int player, Bytes cardHash)
     {
-        Bytes p = players.getDefault(player, Bytes.EMPTY);
+        Bytes p = Players.getDefault(player, Bytes.EMPTY);
 
         if (p == Bytes.EMPTY)
             return; // no such player!
         
-        if (player_cards_1.getDefault(p, Bytes.EMPTY) == Bytes.EMPTY)
+        if (PlayerCards1.getDefault(p, Bytes.EMPTY) == Bytes.EMPTY)
         {
-            player_cards_1.put(p, cardHash);
+            PlayerCards1.put(p, cardHash);
         }
         else
         {
-            player_cards_2.put(p, cardHash);
+            PlayerCards2.put(p, cardHash);
         }
     }
-    public string showdown(string cardSalt, Bytes dealtCards)
+    public string Showdown(string cardSalt, Bytes dealtCards)
     {
-        return _showdown(cardSalt, dealtCards);
+        return showdown(cardSalt, dealtCards);
     }
-    private string _showdown(string cardSalt, Bytes dealtCards)
+    private string showdown(string cardSalt, Bytes dealtCards)
     {
-        byte zero = Convert.ToByte(0);
-        Bytes empty = new Bytes(zero);
-        Bytes defPlayer = players.getDefault(-1, empty);
-        int pcount = defPlayer[0];
-        int dealt = pcount * 2 + table_cards.getDefault(-1, 0);
+        Bytes playersCountB = Players.getDefault(-1, new Bytes(0));
+        int playersCount = playersCountB[0];
+        int dealt = playersCount * 2 + TableCards.getDefault(-1, 0);
         
         if (dealt != Convert.ToInt32(dealtCards[0])) // length
+	    // should be a built-in error
             return "count mismatch! was: " + Convert.ToString(dealt) + " got: " + Convert.ToString(dealtCards[0]); // showdown can't be verified!
         
         for (int i = 0; i < dealt; i++)
         {
             int card = Convert.ToInt32(dealtCards[i + 1]);
-            Bytes h2 = StdLib.Ripemd160(cardSalt + Convert.ToString(card));
-            if (i < (pcount * 2))
+            Bytes cardHash = StdLib.Ripemd160(cardSalt + Convert.ToString(card));
+            if (i < (playersCount * 2))
             {
-                Bytes hash;
-                Bytes p = players.get(i % pcount);
-                if (i < pcount)
-                    hash = player_cards_1.getDefault(p, Bytes.EMPTY);
+                Bytes playerHash;
+                Bytes p = Players.get(i % playersCount);
+                if (i < playersCount)
+                    playerHash = PlayerCards1.getDefault(p, Bytes.EMPTY);
```

Comment:
`playerHash` here is actually a stored player card hash to compare with.

Reply:
oops, indeed